Packs/DuoAdminApi/ReleaseNotes/3_0_2.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### DUO Admin
+#  - Added command get_all_bypass_codes, which provides the ability to pull information from the bypass_code table.
+#     This identifies users in bypass mode, and allows for actions against these accounts
+#  - Added command get_all_admins, which provides the ability to get a list of the administrator accounts
+#  - Added command modify_admin_user, which allows the use of the update_admin command within the api
+#  - Added command modify_user, which allows the use of the update_user command within the api
+#  - Updated command get_all_users to include more of the fields available within the user table
+- Updated the Docker image to: *demisto/duoadmin:1.0.0.24033*.
```

Comment:
It lokks like the docker image doesn't changede.
![Screen Shot 2022-08-01 at 9 05 06](https://user-images.githubusercontent.com/78307768/182082671-adfb1b09-a99e-4581-a6bf-7f382d1eab75.png)